app/Views/errors/html/error_exception.php

```diff
@@ -178,7 +178,7 @@
 							<tr>
 								<td><?= htmlspecialchars($key, ENT_IGNORE, 'UTF-8') ?></td>
 								<td>
-									<?php if (! is_array($value) && ! is_object($value)) : ?>
+									<?php if (! is_array($value) && ! is_object($value) && ! is_resource($value)) : ?>
```

Comment:
1st parameter on htmlspecialchas must be string, how about use `is_string($value)` ?

Reply:
Good point!